packages/extension-ui/src/Popup/Accounts/AddAccount.tsx

```diff
@@ -1,68 +1,62 @@
-// Copyright 2019-2020 @polkadot/extension-ui authors & contributors
```

Comment:
It is fine cloning/adapting, that is what it is all about. Please however do keep copyright headers intact, you can add you own.